### PR TITLE
Release changelog for 2.33

### PR DESCRIPTION
**Pull request by sergiusens**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,68 @@
...
+  * pluginhandler: don't clobber path on local import failure (#1361)
+
+  [ Paolo Pisati ]
+  * core: use the correct kernel arch for s390 (#1376)
+
+  [ Tim Süberkrüb ]
+  * core: support yaml merge tags (#1384)
+
+  [ roxd ]
+  * blank version should not be allowed in snapcraft.yaml (#1379)
```
Comment (kyrofa) [inline debian/changelog]: Perhaps this should be prepended with "schema:" to match the rest?

File: debian/changelog [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+
+  [ Paolo Pisati ]
+  * kernel plugin: add a default target per powerpc, ppc64el and s390x (#1377)
+
+  [ Rex Tsai ]
+  * tests: mock download for rustup in the x-compile test. (#1396)
+
+  [ Evan Dandrea ]
+  * file_utils: handle I/O errors in os.link (#1369)
+  * Correct capitalisation for PyPI. (#1401)
```
Comment (kyrofa) [inline debian/changelog]: This one doesn't match either, including capitalization.

File: debian/changelog [hunk 3]
```diff
@@ -1,3 +1,68 @@
...
+  * pluginhandler: check for collisions only in existing files (#1405)
+  * tests: run the tests in travis using LXD containers (#1415)
+  * tests: build the snapcraft snap in travis tests (#1411)
+  * ci: remove the old script to run with lxc (#1422)
+
+  [ Christian Dywan ]
+  * cli: proper parsing of hidden options (#1378)
+  * core: clean the container on a full clean (#1372)
+  * lxd: do not assume user ID of 1000 for raw.idmap (#1385)
+  * autotools: Enable cross-compilation support (#1383)
```
Comment (elopio) [inline debian/changelog]: `autotools plugin`, to be consistent.

File: debian/changelog (hunk 2 above)
Comment (elopio) [inline debian/changelog]: this could be `python plugin:`

File: debian/changelog (hunk 1 above)
Comment (elopio) [inline debian/changelog]: I'm not sure what section this is about.
Maybe: `validation:`, `schema` or `project loader`.

File: debian/changelog [hunk 4]
```diff
@@ -1,3 +1,68 @@
...
+  * blank version should not be allowed in snapcraft.yaml (#1379)
+
+  [ Paolo Pisati ]
+  * kernel plugin: add a default target per powerpc, ppc64el and s390x (#1377)
+
+  [ Rex Tsai ]
+  * tests: mock download for rustup in the x-compile test. (#1396)
+
+  [ Evan Dandrea ]
+  * file_utils: handle I/O errors in os.link (#1369)
```
Comment (elopio) [inline debian/changelog]: `file utils`, to be consistent.

File: debian/changelog (hunk 3 above)
Comment (sergiusens) [inline debian/changelog]: yeah, I was grabbing my head as I hit merge on this one :-P

File: debian/changelog (hunk 4 above)
Comment (sergiusens) [inline debian/changelog]: all the previous changelog entries say `file_utils` though so I am leaving this as is.

**Pull request merged**